src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,47 @@
+.calendar {
+  margin: 71.5px 149px;
+  font-size: 30px;
+  padding: $square-paddings;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $square-spacing;
```

Comment:
The variable `$square-spacing` is used here but needs to be defined in your SCSS files to avoid compilation errors.